modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -95,7 +95,7 @@
 				    print(get_stack())
 				[/codeblock]
 				Starting from [code]_ready()[/code], [code]bar()[/code] would print:
-				[codeblock]
+				[codeblock lang=text]
```

Comment:
Unless it's derived from somewhere where it's standard, I'd rather be explicit and write the whole parameter name (`languange=text`) instead.
Also there's other places where this would be needed but for this PR it's completely fine to only use it here as an example.

Reply:
> Unless it's derived from somewhere where it's standard, I'd rather be explicit and write the whole parameter name (`languange=text`) instead.

Your typo makes a strong case against this suggestion ;)

`lang` is clear enough and less prone to typos.

Reply:
Yes, this is a follow-up task for documentation team if this PR gets merged. I corrected only one file to test the RST generator and for example.

Reply:
BTW an alternative would be to always use `[codeblocks][gdscript]...` for actual code examples, and thus also make sure we provide C# examples where relevant. And then default to assuming plain text for `[codeblock]`.

Maybe with an optional `[codeblock lang=gdscript]`/`[codeblock lang=csharp]` for whenever it really doesn't make sense to provide both languages (e.g. in `@GDScript` docs).

Reply:
Up the air now, but this addition has me question if the `[codeblocks]` + `[gdscript]`/`[csharp]` syntax should be replaced by `[codeblocks]` + `[codeblock lang=gdscript/csharp]`. Not to break what isn't broke, but just a thought.

Reply:
We literally wrote the same thing within the same minute

Reply:
`[codeblocks]`, `[gdscript]` and `[csharp]` are more like preprocessor directives that depend on the editor setting. While `[codeblock lang=code]` does not depend on the setting. Also hypothetically you could use (this already works):

```
Prints:
[codeblocks]
[gdscript lang=text]
GDScript output.
[/gdscript]
[csharp lang=text]
C# output.
[/csharp]
[/codeblocks]
```

Reply:
> Up the air now, but this addition has me question if the `[codeblocks]` + `[gdscript]`/`[csharp]` syntax should be replaced by `[codeblocks]` + `[codeblock lang=gdscript/csharp]`. Not to break what isn't broke, but just a thought.

Well IMO that would be quite verbose:
```
[codeblocks]
[codeblock lang=gdscript]
pass
[/codeblock]
[/codeblocks]
```
and hard to read, where `s` is the only difference between different tags.

It's a pre-existing issue with us having both `[codeblock]` and `[codeblocks]`, but both are top-level tags, while `[gdscript]`/`[csharp]` are nested tags that further specify `[codeblocks]`.